skeleton/src/main/java/com/dropbox/kaiken/skeleton/skeleton/usermanagement/AuthInfo.kt

```diff
@@ -0,0 +1,34 @@
+package com.dropbox.kaiken.skeleton.skeleton.usermanagement
+
+
+typealias UserId = String
+typealias AccessToken = String
+
+sealed class AuthInfo {
+    data class LoggedOut(val userId: String) : AuthInfo()
+    data class LoggedIn(val userId: String, val oauth2Token: String) : AuthInfo()
+    object NoAuth : AuthInfo()
+}
+
+/**
+ * A holder which gives basic user information from the [UserManager]
+ */
+data class User(val userId: String, val accessToken: String, val isActiveUser: Boolean)
+
+/**
+ * Potential information the [UserManager] can return.
+ */
+data class UserState(
```

Comment:
I think we still need to send which active user changed here.  Otherwise this might emit w/ a list of users, and empty lists in added / removed.

Reply:
added as todo/followup. I'm still not 100% on the api